### PR TITLE
[MIG] mrp_workcenter_account_move: from v8 to v11 task#25067

### PR DESCRIPTION
**Pull request by keylor2906**
[MIG] account_move_line_production: from v8 to v11

[MIG] mrp_rounting_account_journal: from v8 to v11

[MIG] stock_deviation_account: from v8 to v11

### CODE REVIEW

**Comment by keylor2906** [comment]
DUMMY MR for Lodi https://git.vauxoo.com/vauxoo/lodi/merge_requests/41

**Comment by hbto** [comment]
@keylor2906 
Tests must be kept, 
demo data must be kept

Thus TDD approach can be done properly,

any doubt I am here to help you.

Regards.

**Comment by sabrinagnu** [comment]
@hbto may you please check this again (asap please)
CC @keylor2906

File: account_move_line_production/__manifest__.py [hunk 1]
```diff
@@ -11,8 +11,10 @@
         "mrp",
     ],
     "demo": [
-        'demo/demo.xml'
```
Comment (hbto) [inline account_move_line_production/__manifest__.py]: This must be kept thus allowing to use demo data

**Review by hbto — COMMENTED** [review]
There has been no changes since last review.

Last review request for change is still standing.

Regards.

**Comment by keylor2906** [comment]
@hbto 
This [ref](https://github.com/vauxoo-dev/addons-vauxoo/blob/11.0-mig-mrp_workcenter_account_move-dev-keylor29069/account_move_line_production/demo/demo.xml#L9) the referenced data was here on [v8](https://github.com/Vauxoo/odoo/blob/8.0/addons/account/demo/account_minimal.xml#L176) but now in [v11](https://github.com/Vauxoo/odoo/blob/11.0/addons/account/test/account_minimal_test.xml#L112) is not demo data, it's on a file that is only called on tests like [here](https://github.com/Vauxoo/odoo/blob/11.0/addons/account_asset/tests/test_account_asset.py#L17)
could you help me what to do with the demo data.

**Comment by hbto** [comment]
Can we set a meeting for today, write me about 10.00 am (MX) to set the meeting.

Regards.

**Comment by keylor2906** [comment]
@hbto I think this is ready for review

**Comment by keylor2906** [comment]
@hbto Could you review this, please?

**Comment by hbto** [comment]
@keylor2906  Is there any instance where these changes can be tested?

Regards

**Comment by keylor2906** [comment]
@hbto The runbot for the Dummy MR https://git.vauxoo.com/vauxoo/lodi/merge_requests/41 is being regenerated, because it was old, it no longer exist

**Comment by keylor2906** [comment]
@hbto the runbot instance is http://61475-41-958c50.runbot.vauxoo.com

**Comment by hbto** [comment]
Hello @dsabrinarg 

I have tested the "feature" 
I have quoted it not because it is wrong, but because:
- The Feature is creating the Journal Entry with the differences as expected as once depicted [here](https://goo.gl/qFv30X
).
- New Fields/Models were added that is were the quoted section comes. 

@hbto does approve this feature, though a I request that the part regarding to:

mo_00005_-_prod_a_-_routing_for_a_-_odoo

Be thoroughly reviewed to check if the time depicted there are considered in the code properly.

Why does @hbto approve this feature?
Because the main point is to proof that the journal entries are created in the intended way,

I just make a Highlight/Warning to check that the other part (time spent) is being considered.

This code I can see was taken away:
https://github.com/Vauxoo/addons-vauxoo/pull/1289/files#diff-4f88a6aec1e93733f63628f20ca22e33L202

I would like to know if there was another approach taken by Odoo to achieve,
please advice. Depending on that advice we will proceed with the other request @hbto has made.

Regards.

**Pull request merged**